addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16541,3 +16541,13 @@ msgstr ""
 msgctxt "#38016"
 msgid "%d fps"
 msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38120"
+msgid "Support 8 channel DTS HD audio decoding"
+msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38121"
+msgid "This option supports 8 channel DTS HD decoding, but may use more CPU. It is only available when DTS and DTS-HD audio passthrough is disabled"
```

Comment:
@fritsch does this description sound good to you from a technical point or are there errors?
```
Enables conversion of DTS-HD audio streams into a format supported by the connected audio device and will give bitperfect 8 channel audio fidelity on devices that support PCM. Note that this might increase CPU load and is only available when DTS and DTS-HD audio passthrough are disabled.
```
What I wonder though is, why does DTS-HD decoding depend on DTS passthrough? Because DTS-HD does also contain a DTS core stream? What about AVRs that support DTS but not DTS-HD - can't I have DTS passthrough for videos with DTS track but decoding to PCM when I have a DTS-HD track?